index.js

```diff
@@ -35,5 +35,9 @@ module.exports = concurrency => {
 		}
 	};
 
-	return (fn, ...args) => new Promise(resolve => enqueue(fn, resolve, ...args));
+	const generator = (fn, ...args) => new Promise(resolve => enqueue(fn, resolve, ...args));
+	generator.activeCount = () => activeCount;
+	generator.pendingCount = () => queue.length;
```

Comment:
Would be better to make these property getters using `Object.defineProperty()` instead of methods.

Reply:
Done. For the README file, my native language is not english but I believe that I explained it well.